bitcoin/src/blockdata/transaction.rs

```diff
@@ -200,9 +208,6 @@ pub struct TxIn {
 }
 
 impl TxIn {
-    /// The weight of a `TxIn` excluding the `script_sig` and `witness`.
-    pub const BASE_WEIGHT: Weight = Weight::from_wu(32 + 4 + 4);
```

Comment:
I was using this.  Can you leave this in?  There's a [comment](https://github.com/rust-bitcoin/rust-bitcoin/pull/2001#issuecomment-1710526335) by @stevenroose which is correct in saying this should be multiplied by 4.

Reply:
No, "base size" is well defined term in the bip. I should never have acked this const, its badly named, badly documented, and wrong. If you want this as a const you can use `Weight::from_vb(Sequence::SIZE + OutPoint::SIZE)` (or `from_vb_uncheked` if you want const).

Reply:
>  I should never have acked this const

You do a lot of code review which is a thankless job most of the time and helps the project a lot.  And when one of the many you reviewed has a mistake then people notice your review.  There are a lot of maintainers on this project that never bother to code review as far as I can tell.  Really I'm not sure what the point is of having lots of maintainers that never code review.